cf-macro/index.js

```diff
@@ -0,0 +1,83 @@
+'use strict'
+
+const _ = require('lodash')
+const Ajv = require('ajv')
+
+const alarms = require('slic-watch-core/alarms')
+const dashboard = require('slic-watch-core/dashboard')
+const CloudFormationTemplate = require('slic-watch-core/cf-template')
+const defaultConfig = require('slic-watch-core/default-config')
+const { slicWatchSchema } = require('slic-watch-core/config-schema')
+
+exports.handler = async function (event, context) {
+  const response = processFragment(event)
+  return response
+}
+
+function processFragment (event) {
+  let status = 'success'
+  let outputFragment = event.fragment
+  try {
+    const slicWatchConfig = (outputFragment.Metadata || {}).slicWatch || {}
+    const ajv = new Ajv({
+      unicodeRegExp: false
+    })
+    const config = _.merge(defaultConfig, slicWatchConfig)
+
+    const slicWatchValidate = ajv.compile(slicWatchSchema)
+    const slicWatchValid = slicWatchValidate(slicWatchConfig)
+
+    if (!slicWatchValid) {
+      throw new Error('SLIC Watch configuration is invalid: ' + ajv.errorsText(slicWatchValidate.errors))
+    }
```

Comment:
A DX question here: should we allow users to be able to disable a SLIC Watch entirely and even ignore configuration errors?

Maybe a user is playing around with some new config (or with SLIC Watch for the first time) and for whatever reason they just want to disable it while they test things.

Should disabling SLIC Watch also stop us from validating that the configuration is correct?

If we think that's the case we could simply move lines 28-32 inside the following `if` statement and validate the configuration only if SLIC Watch is actually enabled.

Reply:
Sure, moved it.